pom.xml

```diff
@@ -31,15 +31,12 @@
     <properties>
         <revision>2.0.3</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.60.3</jenkins.version>
+        <jenkins.version>2.164.3</jenkins.version>
         <java.level>8</java.level>
-        <jenkins-test-harness.version>2.13</jenkins-test-harness.version>
-        <jackson.version>2.10.1</jackson.version>
-        <jetty.version>9.2.15.v20160210</jetty.version>
-
         <azuresdk.version>1.31.0</azuresdk.version>
         <azure-commons.version>1.0.4</azure-commons.version>
         <azure-keyvault.version>1.0.0</azure-keyvault.version>
+        <hpi.compatibleSinceVersion>2.1</hpi.compatibleSinceVersion>
```

Comment:
there's a breaking change in the new SDK where tenant ID is required for all credentials, I wasn't able to find a way to not set the tenant ID at all.

There was some code that allowed using a username/password credential, but it appears not to be documented anywhere and unlikely to be used.

This is also a big breaking change for the azure-keyvault plugin, but I would rather release together than try make it compatible, I have a companion PR here:
https://github.com/jenkinsci/azure-keyvault-plugin/pull/56